java/cli/build.gradle

```diff
@@ -1,26 +1,25 @@
 plugins {
     id 'application'
     id 'eclipse'
-    id 'java'
 }
 
 group = 'ome.jxrlib-cli'
 version = '0.2.2'
 
 mainClassName = 'ome.jxrlib.Main'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 repositories {
     jcenter()
-    //mavenLocal()
+    mavenLocal()
```

Comment:
I expect this should be the main change required to let https://ci.openmicroscopy.org/view/Mgmt/job/JXRLIB-build pass again as `./gradlew -b java/cli/build.gradle clean installDist distZip` should be able to consume the `jxrlib-all` artifact built locally.